spec/Section 3 -- Type System.md

```diff
@@ -1473,7 +1474,7 @@ EnumValuesDefinition : { EnumValueDefinition+ }
 EnumValueDefinition : Description? EnumValue Directives[Const]?
 
 GraphQL Enum types, like Scalar types, also represent leaf values in a GraphQL
-type system. However Enum types describe the set of possible values.
+type system. However Enum types describe the list of possible values.
```

Comment:
I considered "However Enum types describe as a list the set of possible values." but it didn't seem worth the extra words.